### PR TITLE
Cannot install connexion when using markupsafe>=2.1.0

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related issues or additional information of the supplied change

Related to https://github.com/thoth-station/user-api/pull/1681#issuecomment-1049085925

## Description

Prescription for installing `markupsafe <2.1.0` with `connexion`

### CODE REVIEW

File: prescriptions/co_/connexion/markupsafe_210.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+units:
+  steps:
+  - name: ConnexionMarkupsafe210Step
+    type: step
+    should_include:
+      adviser_pipeline: true
+    match:
+      package_version:
+        name: connexion
+        version: "*"
```
[the remark this answers is not in the document]
Comment (fridex) [reply prescriptions/co_/connexion/markupsafe_210.yaml:10]: If not provided, any version is matched.

File: prescriptions/co_/connexion/markupsafe_210.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+units:
+  steps:
+  - name: ConnexionMarkupsafe210Step
+    type: step
```
[the remark this answers is not in the document]
Comment (mayaCostantini) [reply prescriptions/co_/connexion/markupsafe_210.yaml]: Thanks, changes were made in https://github.com/thoth-station/prescriptions/pull/19748/commits/2c96582fbbbf611e0bb6e1b93f69085f543aa6bb 💯

File: prescriptions/co_/connexion/markupsafe_210.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - group:
+      - package_version:
+        name: connexion
+        index_url: https://pypi.org/simple
+      - package_version:
+        name: markupsafe
+        version: ">=2.1.0"
+        index_url: https://pypi.org/simple
+    run:
+      multi_package_resolution: true
```
Comment (fridex) [inline prescriptions/co_/connexion/markupsafe_210.yaml:17]: This option is not available in the group step:

```suggestion
```

File: prescriptions/co_/connexion/markupsafe_210.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+units:
+  steps:
+  - name: ConnexionMarkupsafe210Step
+    type: step.Group
+    should_include:
+      adviser_pipeline: true
+    match:
+    - group:
+      - package_version:
+        name: connexion
```
Comment (fridex) [inline prescriptions/co_/connexion/markupsafe_210.yaml:10]: Here, `name` and `index_url` should be part of `package_version` entry (the indentation is wrong).

File: prescriptions/co_/connexion/markupsafe_210.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        # Filter connexion if Python container images are
+        # shipped with the specified markupsafe.
+        python_packages:
+        - name: markupsafe
+          version: ">=2.1.0"
+          location: ^/opt/app-root/.*
+    match:
+      package_version:
+        name: connexion
+        version: "*"
```
Comment (fridex) [inline prescriptions/co_/connexion/markupsafe_210.yaml:39]:
```suggestion
```

**Review by fridex — COMMENTED** [review]
By the way, you can point adviser to prescriptions for validating  - something like:

```
pipenv run python3 ./thoth-adviser validate-prescriptions ../prescriptions/prescriptions
```

adviser should report back any issues with the validation.

File: prescriptions/co_/connexion/markupsafe_210.yaml (hunk 4 above)
Comment (fridex) [reply prescriptions/co_/connexion/markupsafe_210.yaml:10]: Also below.

**Comment by mayaCostantini** [comment]
Thanks for the review @fridex, the prescription was validated successfully for new changes

**Pull request merged**